fastlane/Fastfile

```diff
@@ -620,6 +620,9 @@ end
 
     set_info_plist_value(path: "ownCloud Action Extension/Info.plist", key: "CFBundleDisplayName", value: "Save to " + appName)
     set_info_plist_value(path: "ownCloud Action Extension/Info.plist", key: "CFBundleName", value: appName)
+    
+    sh("sed -i '' '/\"CFBundleDisplayName\" =/s/ownCloud/#{appName}/' ownCloud Action Extension/en.lproj/InfoPlist.strings")
+    sh("sed -i '' '/\"CFBundleDisplayName\" =/s/ownCloud/#{appName}/' ownCloud Action Extension/de.lproj/InfoPlist.strings")
```

Comment:
I didn't have a chance to test this out, but I could imagine that this replacement fails because the spaces in `ownCloud Action Extension` in the path are not escaped, so that `sed` sees it as three paths (`ownCloud`, `Action`, `Extension`) instead.

```suggestion
    sh("sed -i '' '/\"CFBundleDisplayName\" =/s/ownCloud/#{appName}/' ownCloud\ Action\ Extension/en.lproj/InfoPlist.strings")
    sh("sed -i '' '/\"CFBundleDisplayName\" =/s/ownCloud/#{appName}/' ownCloud\ Action\ Extension/de.lproj/InfoPlist.strings")
```